harmoni_dialogues/harmoni_bot/src/rasa_example/addons/custom_channel.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+import inspect
+from sanic import Sanic, Blueprint, response
```

Comment:
Is this in our dependencies somewhere or is it installed with some other dependency?

Reply:
All of the Rasa dependencies are installed with `pip install rasa`.